alephclient/api.py

```diff
@@ -97,38 +68,44 @@ def _request(self, method, url, **kwargs):
 
         if len(response.text):
             return response.json()
+        else:
+            # NOTE: Raise exception?
+            return {}
```

Comment:
I think we should raise an exception there.

Reply:
Yes, maybe when HTTP > 399?

Reply:
We already raise an exception in that case (`response.raise_for_status()`).

I'm not sure in which situation `response.text` can be empty *and* the response code <400 but again, mypy really dislikes the cases where you have no `return`.

`raise AlephException(exc)` would do the trick, I think.